pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1637,10 +1639,9 @@ private void createPersistentTopic(final String topic, boolean createIfMissing,
                                        Map<String, String> properties, @Nullable TopicPolicies topicPolicies) {
         TopicName topicName = TopicName.get(topic);
         final long topicCreateTimeMs = TimeUnit.NANOSECONDS.toMillis(System.nanoTime());
-
         topicFuture.exceptionally(t -> {
             pulsarStats.recordTopicLoadFailed();
-            return null;
+            return Optional.empty();
```

Comment:
This change is not necessary since the stage returned by `exceptionally()` is not being used